shortcodes/ucf-faq-search-shortcode.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * Provides a shortcode for the UCF FAQ Search
+ */
+if ( ! class_exists( 'UCF_FAQ_Search_Shortcode' ) ) {
+	class UCF_FAQ_Search_Shortcode {
+		public static function shortcode( $atts ) {
+			$atts = shortcode_atts( array(
+				'title'         => 'Search All FAQs',
+				'title_tag'     => 'h2',
+				'title_classes' => 'h3 text-secondary',
+				'placeholder'   => 'Search&hellip;'
+			), $atts );
+
+			$title         = $atts['title'];
+			$title_tag     = $atts['title_tag'];
+			$title_classes = $atts['title_classes'];
+
+			ob_start();
+		?>
+			<<?php echo $title_tag; ?> class="<?php echo $title_classes; ?>"><?php echo $title; ?></<?php echo $title_tag; ?>>
```

Comment:
We likely need to either ensure the `input` here has an `aria-labelledby` attribute pointing back to the title wrapper elem, or ensure a `label` element is used instead.